pkg/securitypolicy/securitypolicy_test.go

```diff
@@ -300,7 +312,10 @@ func Test_EnforceOverlayMountPolicy_Multiple_Instances_Same_Container(t *testing
 			containers = append(containers, c)
 		}
 
-		sp := NewStandardSecurityPolicyEnforcer(containers, "")
+		sp, err := NewStandardSecurityPolicyEnforcer(containers, "")
+		if err != nil {
+			t.Fatalf("failed to create standard enforcer: %s", err)
+		}
```

Comment:
how is it possible for this to error now? that seems bad that it can error.

Reply:
I dislike the idea of introducing a mechanism that makes this function partial unless it is absolutely neccessary. Functions that do not have an error case are in my mind, a goal to aim for assuming the tradeoffs aren't awful.